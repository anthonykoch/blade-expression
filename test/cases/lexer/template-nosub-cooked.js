`\b \f \n \r \t \v \{ \a \u{1f4ac}\u0020\u{20}\x20`
