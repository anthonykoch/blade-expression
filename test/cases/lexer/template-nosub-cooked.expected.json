---
name: Template - no substitution cooked
description: no sub templated cooked
skip: true
---

[
  {"type": 10, "value": "`\b \f \n \r \t \u000b { a ðŸ’¬   `", "line": 1, "column": 0, "start": 0, "end": 23 }
]
